FSW/src/headers/config.hpp

```diff
@@ -69,13 +80,45 @@ static Event saveBufferEvent;
 static TimedEvent sunriseTimerEvent(WINDOW_LENGTH_MSEC);
 static TimedEvent sweepTimeoutEvent(SWEEP_TIMEOUT_MSEC);
 
+// FUTURE TEAMS: this event is invoked when the ADCS should switch its sweep direction
+//   so link your ADCS module with this event to tell it when to switch direction 
+//      look at checkSweepChange() in timing.cpp for more info
+static Event sweepDirectionChangeEvent; 
+
+
 /* - - - - - - Command Handling Module - - - - - - */
 const int COMMAND_QUEUE_SIZE = 100;     // maximum number of commands the command queue can store.
 const int SERIAL_TIMEOUT_MSEC = 50;     // milliseconds, maximum time to wait for serial input
 
+
 /* - - - - - - Fault Mitigation Module - - - - - - */
-const int WD_RESET_INTERVAL = 100;  // watchdog feeding interval, ms
-const int WD_PULSE_DUR = 10;        // watchdog reset signal duration, MICROSECONDS!!!
+const int WD_RESET_INTERVAL_MSEC = 100;     // milliseconds, watchdog feeding interval
+const int WD_PULSE_DUR_MICROSEC = 10;       // microseconds, watchdog reset signal duration
+
+
+/* - - - - - - Housekeeping Module - - - - - - */
+const int HK_SAMPLES_TO_KEEP = 100;   // number of previous housekeeping samples to keep
+
+// heater cutoff temperatures
+const float HEATER_TEMP_LOW = 0;     // celsius, heater will turn on at or below this temp

```

Comment:
I think Danny was saying something about the fact that the heater should basically always be on. Probably still good to have the upper limit though. 

I would design the heater to be always on by default, but I don't really like the idea of that for testing, draws a lot of power.

Reply:
I added a bool `forceHeaterOn` that `setHeater()` will check first, and if true will just force the heater on all the time.